### DISCUSSION TITLE
Request for simplified menu... no drop down items to save clicks

### DISCUSSION THREAD

**Discussion by StoneEngineer** [Kovah/LinkAce#537] [Ideas & Feature Requests]
Hi Kevin,

First, thanks for great bookmarking tool. Installation and DB configuration was smooth.

I would be happy to have simplified browser menu without dropdown items, if you don't mind, to save clicks.
The menu items **Add Link** and **All Links** are already there.
There is enough space to have them all:
**Add Link** **All Links** **Add List** **All Lists** **Add Tag** **All Tags**

Same in mobile browser bottom menu. There is space for 4 separate icons.

Stone

**Comment by scomoletti** [comment]
To go along with this idea, simplify the link search as well. Instead of an icon to search links, which goes to the 'advanced search' page, just have an input box which searches by title and description. Then have an icon for 'advanced' search functionality next to the input box which displays the current search page.

**Comment by Kovah** [comment]
Will be solved in LinkAce 2.0 which will be out in the near future. See #553.
